pkg/utils/image/infos_test.go

```diff
@@ -3,7 +3,8 @@ package image
 import (
 	"testing"
 
-	"github.com/stretchr/testify/assert"
+	"gotest.tools/assert"
+	is "gotest.tools/assert/cmp"
```

Comment:
Curios what these additional/different imports are and where they are used?

Reply:
Why the switch from testify to gotools?

Reply:
Don't know, I copied those from another test suite... I can revert to testify if you want.

Reply:
I don't have any personal preference. Would you recommend one over the other ?

Reply:
Going to change it to use testify.
Out of curiosity, any reason why testify is better ?

Reply:
Fixed.